docs/EN_US/Installing_and_RunningTheHPCCPlatform/Inst-Mods/ZAP.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE article PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<book>
```

Comment:
Does this included module need to be a <book>? It seems like this contains a significant amount of superfluous content.